### ISSUE TITLE
Why is "info" lower than "debug" by default?

### ISSUE THREAD

**Issue by levity** [balderdashy/captains-log#13]
This is the opposite of what I expected, having worked with systems that treat "debug" as more verbose and lower than "info" for my entire career.

**Comment by danielgranat** [comment]
Curious to find the answer to this as well.
Seams odd that i need to change all my debug messages to informative just to i don't see those in production.

**Comment by sailsbot** [comment]
Thanks for posting, @levity.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
